R/print.R

```diff
@@ -4,11 +4,18 @@
 #' @param x The result object.
 #' @param ... Ignored.
 #' @return The JSON result.
-#' 
+#'
 #' @importFrom jsonlite prettify toJSON
 #' @export
 #' @method print gh_response
 
 print.gh_response <- function(x, ...) {
-  print(toJSON(unclass(x), pretty = TRUE, auto_unbox = TRUE))
+  if("raw" %in% class(x)) {
```

Comment:
Can you pls put a space after the `if`?

Reply:
And also, just use `inherits(x, "raw")`?

Reply:
Done